independent-projects/bootstrap/pom.xml

```diff
@@ -49,10 +49,10 @@
         <eclipse-minimal-json.version>0.9.5</eclipse-minimal-json.version>
         <jboss-logging.version>3.5.0.Final</jboss-logging.version>
         <junit.jupiter.version>5.9.2</junit.jupiter.version>
-        <maven-core.version>3.8.7</maven-core.version><!-- Keep in sync with sisu.version -->
+        <maven-core.version>3.9.0</maven-core.version><!-- Keep in sync with sisu.version -->
```

Comment:
Given that 3.9.1 is about to be released and contains some rather important fixed, I'd personally suggest to skip 3.9.0.

Reply:
No problem skipping 3.9.0.

Reply:
3.9.1 is out, https://maven.apache.org/docs/3.9.1/release-notes.html